### PR TITLE
CombineNestedDirnameFixer - Add space after comma

### PR DESCRIPTION
**Pull request by julienfalque**
Ref: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3475#discussion_r331811880.

### CODE REVIEW

File: src/Fixer/FunctionNotation/CombineNestedDirnameFixer.php [hunk 1]
```diff
@@ -225,7 +225,7 @@ private function combineDirnames(Tokens $tokens, array $dirnameInfoArray)
             $prev = $tokens->getPrevMeaningfulToken($outerDirnameInfo['end']);
             $items = [];
             if (!$tokens[$prev]->equals(',')) {
-                $items[] = new Token(',');
+                $items = [new Token(','), new Token([T_WHITESPACE, ' '])];
```
Comment (keradus) [inline src/Fixer/FunctionNotation/CombineNestedDirnameFixer.php]: here, we add missing space to calling `dirname`, maybe we should that feature regardless of which method is called?

like we have `./php-cs-fixer describe function_declaration`, maybe we should have `function_invocation` ?

Comment (keradus) [reply src/Fixer/FunctionNotation/CombineNestedDirnameFixer.php]: i read comments you ref to, nevermind

**Comment by keradus** [comment]
should we drop the priority relation and move `tests/Fixtures/Integration/priority/combine_nested_dirname,method_argument_space.test` test from `priority` to `misc` , @julienfalque ?

**Comment by julienfalque** [comment]
No, the priority test still fails if `combine_nested_dirname` runs after `method_argument_space`.

**Comment by SpacePossum** [comment]
Thank you @julienfalque.

**Pull request merged**